src/utils/logic-app/templateUtils.ts

```diff
@@ -64,21 +65,37 @@ export function generateDeploymentTemplateParameters(parameters: Record<string,
 }
 
 export function generateParameters(workflow: Workflow) {
+    let parameters =  Object.assign({}, workflow.definition.parameters);
+
+    for (var key in parameters) {
```

Comment:
`for (const key of Object.keys(parameters) {`

Reply:
That what I thought (and tried) first, but weirdly it doesn't work - it returns [0,1, 2] instead...?

Reply:
`parameters` should be an object, not an array.  What does `parameters` look like?

Reply:
It is an object, but somehow I was only able to access its keys via `for(var x in parameters)` not `for(var x in Object.keys(parameters)`. Maybe I did something wrong, will double check... 

```
"parameters": {
   "foo": {},
   "bar": {}
}
```

Reply:
That should be `for..of`, not `for..in`.

Reply:
Ah!

Reply:
Fixed pending udpate